docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -0,0 +1,231 @@
+of the colormap. Any photon count higher than that will be clipped to the maximal color.
+
+What we can nicely see in this data is that as more force is applied, we get an increased binding activity. Let’s see
+if we can put the kymotracker to some good use and quantify these.
+
+Downsampling the kymograph
+--------------------------
+
+To make it a bit easier to tweak the algorithm parameters, we will make use of a notebook widget. While we could work
+on the full time resolution data, we can make things a little easier for the kymotracking algorithm by downsampling the
+data a little bit. Here we extract the data corresponding to the green channel using the `kymo` attribute
+`green_image` and downsample the image by a factor of `2` using
+`block_reduce() <https://scikit-image.org/docs/dev/api/skimage.measure.html#skimage.measure.block_reduce>`_ from
+`scikit image`. This function takes an image and downsamples it according to its second argument. Here the downsampling
+factor is given for each dimension. In our case we have two dimensions: position and time. We only want to downsample
+time and not position, so we provide `(1, downsample_factor)`::
+
+    data = kymo.green_image
+    downsample_factor = 2
+    data = block_reduce(data, (1, downsample_factor))
+
+Given that we have downsampled the data, we should also use a time axis that takes into account that we have
+downsampled our data. We can calculate the time step per line using the line time found in `Kymo`::
+
+    dt = downsample_factor * kymo.line_time_seconds
+
+The physical size of a pixel along the line we can obtain from the `json` field of the `Kymo`::
+
+    pixel_size = kymo.pixelsize_um
+
+Performing the kymotracking
+---------------------------
+
+Now that we’ve loaded some data, we can begin tracing lines on it. For this, we open the KymoWidget we’ve imported.
+This will provide us with a view of the kymograph and some dials to tune in algorithm settings. We open with a
+custom `axis_aspect_ratio` which determines our field of view. This input is not necessary, but provides a better
+view of our data.
+
+We will be using the greedy algorithm. The `threshold` should typically be chosen somewhere between the expected
+baseline photon count and the photon count of a true line (note that you can see the local photon count between square
+brackets while hovering over the kymograph). The `line width` should roughly be set to the expected line width of a
+line. The `window` should be chosen such that small gaps in a trace can be overcome, but not so large that spurious
+points may be strung together as a trace. `Sigma` controls how much the location can fluctuate from one point to the
+next, while the `min length` determines how many peak points should be in a trace for it to be considered a valid
+trace.
+
+Holding down the left mouse button and dragging pans the view, while the outer mouse button allows us to drag a region
```

Comment:
What is the 'outer mouse button'?

Reply:
It's a mistake I often make. Changed it to `right mouse button` as I think that's the more accepted term for this button.

I don't know when I picked up this habit of calling the rmb, omb. Outer is defined with respect to the center of the person. Outer mouse may be more correct though, since left-handed people have the "right click" on the left. 🤪